content/_components/shopify-admin-v2/index.md

```diff
@@ -258,17 +261,17 @@ Input metadata will represent all needed fields to update an object
 
 ### Execute mutation
 
-Execute any mutation available on selected API version. This action can be used to `Create`, `Update` or `Delete` Objects and any other operations that affect on Shopify data
+Execute any mutation available on selected API version. This action can be used to `Create`, `Update` or `Delete` Objects and any other operations that affect on Shopify data.
 
 #### Configuration Fields
 
 * **Mutation type** - (dropdown, required): Mutation type to execute. E.g `Customer Create`.
-* **Select basic fields for resulting object** - (dropdown, optional): Here provided only basic fields that can be included in resulting object, it may affect on query cost
-* **You can provide additional fields here** - (string, optional): Resulting object can be expanded using GraphQL request, it may affect on query cost
+* **Select basic fields for resulting object** - (dropdown, optional): Here provided only basic fields that can be included in resulting object, it may affect on query cost.
+* **You can provide additional fields here** - (string, optional): Resulting object can be expanded using GraphQL request, it may affect on query cost.
```

Comment:
* **You can provide additional fields here** - (string, optional): The resulting object can be expanded using GraphQL request, it may affect query cost.